packages/logs/src/boot/startLogs.ts

```diff
@@ -102,25 +102,35 @@ export function buildAssemble(
   configuration: LogsConfiguration,
   reportError: (error: RawError) => void
 ) {
-  const errorRateLimiter = createEventRateLimiter(StatusType.error, configuration.maxErrorsPerMinute, reportError)
+  const logRateLimiters = {
+    [StatusType.error]: createEventRateLimiter(StatusType.error, configuration.maxErrorsPerMinute, reportError),
+    [StatusType.warn]: createEventRateLimiter(StatusType.warn, configuration.maxWarningsPerMinute, reportError),
+    [StatusType.info]: createEventRateLimiter(StatusType.info, configuration.maxInfosPerMinute, reportError),
+    [StatusType.debug]: createEventRateLimiter(StatusType.debug, configuration.maxDebugsPerMinute, reportError),
+  }
```

Comment:
Indeed we can. If these limiters are only to prevent edge cases, maybe we could simplify both RUM and Logs limiters by having a single limiter for each with the option `maxEventsPerMinute`.

- In Logs it will check all logs whatever the status
- In RUM it will check all events whatever the type

Maybe I'm going too far :). Let me know what you think?